factory/__main__.py

```diff
@@ -57,43 +84,55 @@ def fromTui(console):
     )
 
     image_paths = [
-        os.path.expanduser(path) for path in console.input(
+        expanduser(path)
+        for path in console.input(
             "🖼️ [bold yellow]Which image(s) do you want to manufacture? (image paths separated by spaces):[/] "
         ).split()
     ]
-    palette = console.input("🎨 [bold yellow]Which palette do you prefer? (panther 'pink' or snoopy 'white'):[/] ")
-    if palette not in ['pink', 'white']:
+    palette = console.input(
+        "🎨 [bold yellow]Which palette do you prefer? (panther 'pink' or snoopy 'white'):[/] "
+    )
+    if palette not in ["pink", "white"]:
         console.print(
-                f"❌ [red]We had a problem in the pipeline! \nThe palette you chose is not available! \nShuting down... [/]"
-            )
+            f"❌ [red]We had a problem in the pipeline! \nThe palette you chose is not available! \nShuting down... [/]"
+        )
         palette = None
 
     return (image_paths, palette)
 
+
 def process_image(image_path, console, gruvbox_factory):
     image = gruvbox_factory.open_image(image_path)
 
-    console.print(f"🔨 [yellow]manufacturing '{os.path.basename(image_path)}'...[/]")
+    console.print(f"🔨 [yellow]manufacturing '{basename(image_path)}'...[/]")
 
     # TODO: might be a better idea to save the new Image in the same directory the command is being run from
-    save_path = os.path.join(
-        os.path.dirname(image_path), "gruvbox_" + os.path.basename(image_path)
-    )
+    # NOTE: this is done with the following code:
```

Comment:
I kinda regret this idea, and I think it's better to generate in the same folder the pic is, as it is now. Could you remove this TODO? Let's keep the old way :)